### PR TITLE
[DOC] Fix dashboards management navigation changes and make copy edits to c…

### PR DESCRIPTION
**Pull request by vagimeli**
…ontent

### Description
Fix Dashboard Management navigation menu changes for 2.9 and copy edit content

Screenshot 2023-07-31 at 2 36 04 PM

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/dev-tools/index-dev.md [hunk 1]
```diff
@@ -7,8 +7,14 @@ has_children: true
 
 # Dev Tools
 
-Use **Dev Tools** to set up your OpenSearch Dashboards environment, identify and fix bugs, and customize dashboards' appearance and behavior. 
+Interact directly with OpenSearch by using **Dev Tools** to set up your OpenSearch Dashboards environment, run queries, explore data, and debug problems. To access the Dev Tools console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. The following are examples of how you can use the Dev Tools console in OpenSearch Dashboards: 
 
-To access the Dev Tools console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. You'll see an interface like the one shown in the following image.
+- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication setting for your OpenSearch Dashboards instance.
```
Comment (cwillum) [inline _dashboards/dev-tools/index-dev.md:12]:
```suggestion
- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 
```

File: _dashboards/dev-tools/index-dev.md [hunk 2]
```diff
@@ -7,8 +7,14 @@ has_children: true
...
+Interact directly with OpenSearch by using **Dev Tools** to set up your OpenSearch Dashboards environment, run queries, explore data, and debug problems. To access the Dev Tools console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. The following are examples of how you can use the Dev Tools console in OpenSearch Dashboards: 
 
-To access the Dev Tools console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. You'll see an interface like the one shown in the following image.
+- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication setting for your OpenSearch Dashboards instance. 
+- [Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/). For example, you can use the console to run a query to find all the documents in your index that have a specific word in them.
+- Debug problems with your queries. For example, if your query is not returning the results you expect, you can use the console to see the error message and figure out the problem.
```
Comment (cwillum) [inline _dashboards/dev-tools/index-dev.md:14]:
```suggestion
- Debug problems with your queries. For example, if your query is not returning the results you expect, you can use the console to look for error messages and figure out the problem.
```

**Review by cwillum — APPROVED** [review]
LGTM

File: _dashboards/dev-tools/index-dev.md [hunk 3]
```diff
@@ -7,8 +7,14 @@ has_children: true
...
+- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 
+- [Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/). For example, you can use the console to run a query to find all the documents in your index that have a specific word in them.
+- Debug problems with your queries. For example, if your query is not returning the results you expect, you can use the console to look for error messages and figure out the problem.
+- Learn about the APIs in OpenSearch. The API reference documentation is available in the console under the help button, so you can look up the syntax for different API calls.
+- Develop custom visualizations. For example, you can use the console to create Vega visualizations. 
+- Customize the appearance and behavior of dashboards. For example, you can use the console to customize the colors of visualizations on a dashboard or to add new filters. 
+- Identify and fix bugs. For example, you can use the console to see the logs and try to identify the cause of the problem.
```
Comment (natebower) [inline _dashboards/dev-tools/index-dev.md:18]:
```suggestion
- Identify and fix bugs. For example, you can use the console to view logs and identify the cause of the problem.
```

File: _dashboards/im-dashboards/index.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ The Index Management interface in OpenSearch Dashboards provides a unified solut
 ## Get started with index management using Dashboards
 
 **Step 1: Open Index Management** 
-Once you're in OpenSearch Dashboards, select **Index Management** under the Management main menu. Then select **Indices**.
+Once you're in OpenSearch Dashboards, select **Index Management** under the **OpenSearch Plugins** main menu. Then select **Indices**.
```
Comment (natebower) [inline _dashboards/im-dashboards/index.md:19]:
```suggestion
Once you're in OpenSearch Dashboards, select **Index Management** from the **OpenSearch Plugins** main menu. Then select **Indices**.
```

File: _dashboards/im-dashboards/notifications.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ In the **Notification settings** interface, you can configure the default notifi
 - Clone
 - Force merge
 
-To get started, under the OpenSearch Dashboards main menu, select **Management** > **Index Management**. Under **Index Management**, select **Notification settings**. 
+To get started, under the OpenSearch Dashboards main menu, select **OpenSearch Plugins** > **Index Management**. Under **Index Management**, select **Notification settings**.
```
Comment (natebower) [inline _dashboards/im-dashboards/notifications.md:23]:
```suggestion
To get started, from the OpenSearch Dashboards main menu, select **OpenSearch Plugins** > **Index Management**. Under **Index Management**, select **Notification settings**. 
```

**Review by natebower — COMMENTED** [review]
@vagimeli Just a few minor changes. Thanks!

File: _dashboards/dev-tools/index-dev.md [hunk 6]
```diff
@@ -7,8 +7,14 @@ has_children: true
...
 
-To access the Dev Tools console, select **Dev Tools** under the **Management** menu on the OpenSearch Dashboards home page. You'll see an interface like the one shown in the following image.
+- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 
+- [Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/). For example, you can use the console to run a query to find all the documents in your index that have a specific word in them.
+- Debug problems with your queries. For example, if your query is not returning the results you expect, you can use the console to look for error messages and figure out the problem.
+- Learn about the APIs in OpenSearch. The API reference documentation is available in the console under the help button, so you can look up the syntax for different API calls.
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _dashboards/dev-tools/index-dev.md:15]: Revised.

**Pull request merged**